src/main/resources/edu/cornell/kfs/coa/document/workflow/SubObjectCodeGlobalEditMaintenanceDocument.xml

```diff
@@ -19,7 +19,7 @@
     <documentType>
       <name>GSOI</name>
       <parent>COA</parent>
-      <label>Global Sub-Object Code Inactivation</label>
+      <label>Sub-Object Code Global Inactivation</label>
```

Comment:
This appears to be a change to a doc type instead of a change to the dashboard JSON. Does it still influence the appearance on the KFS dashboard? Or is it an extra fix that's still within the scope of this change?